src/js/controller/FramesListController.js

```diff
@@ -213,6 +215,17 @@
 
   /**
    * @private
+   */
+  ns.FramesListController.prototype.onSortableStop_ = function (event, ui) {
+    this.justDropped = true;
+
+    this.resizeTimer = window.setTimeout($.proxy(function() {
```

Comment:
We don't reuse/clear this timer at any point so no need to store it. And resizeTimer is not really appropriate here :)